src/sampling.jl

```diff
@@ -35,11 +36,11 @@ function Euler_Maruyama_sampler(model::CliMAgen.AbstractDiffusionModel,
         batch_time_step = fill!(similar(init_x, size(init_x)[end]), 1) .* time_step
         g = CliMAgen.diffusion(model, batch_time_step)
         if forward
-            x .= x .+ sqrt(Δt) .* CliMAgen.expand_dims(g, 3) .* randn!(rng, z)
+            x .= x .+ sqrt(Δt) .* CliMAgen.expand_dims(g, nspatial+1) .* randn!(rng, z)
```

Comment:
is this because we broadcast over batch members, or over channels? im a little confused that `x` has nspatial+2 dimensions in this case, but `g` has nspatial+1.

Reply:
ok, I think this makes sense. we go from something that is zero-d to something with nspatial+1, then broadcast over batch.

Reply:
exactly!